coding-concepts.js

```diff
@@ -7,49 +7,49 @@
 // There is no need to change any of the code.
 
 // --------------------INSTRUCTOR EXAMPLE: What will this log?
-const colors = ["tangerine", "magenta", "lilac", "daffodil"]
+// const colors = ["tangerine", "magenta", "lilac", "daffodil"]
 // console.log(colors.push("indigo"))
 
 // a) Your answer:
 // b) Verify and explain:
 
 // --------------------1) What will this log?
 
-const cohort = "LEARN 2023"
+// const cohort = "LEARN 2023"
 // console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// // a) Your answer: 10
+// // b) Verify and explain: Correct because the string has 10 letters and length isn't checking for the number of indexes.
 
-// --------------------2) What will this log?
+// // --------------------2) What will this log?
 
-const greeting = "Hello World!"
+// const greeting = "Hello World!"
 // console.log(greeting[4])
 
-// a) Your answer:
-// b) Verify and explain:
+// // a) Your answer:4
+// // b) Verify and explain:Correct b/c the 4 passed in the square brackets indicates to print the value in the 4th index.
 
-// --------------------3) What will this log?
+// // --------------------3) What will this log?
 
-const languages = ["JavaScript", "Ruby", "Python", "C++"]
-const index = 1
-// console.log(languages[index])
+// const languages = ["JavaScript", "Ruby", "Python", "C++"]
+// const index = 1
+// // console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// // a) Your answer:Javascript
+// // b) Verify and explain:Incorrect b/c I forgot that the square brackets is telling the output to be the first index and not first variable.
 
-// --------------------4) What will this log?
+// // --------------------4) What will this log?
 
-const weekendDays = ["saturday", "sunday"]
-// console.log(weekendDays.toUpperCase())
+// const weekendDays = ["saturday", "sunday"]
+// // console.log(weekendDays.toUpperCase())
 
-// a) Your answer:
-// b) Verify and explain:
+// // a) Your answer: "SATURDAY", "SUNDAY"
+// // b) Verify and explain:typeError
 
-// --------------------5) What will this log?
+// // --------------------5) What will this log?
 
-const dataTypes = ["number", "string", "Boolean", "undefined"]
-// console.log(typeof dataTypes.length)
+// const dataTypes = ["number", "string", "Boolean", "undefined"]
+// // console.log(typeof dataTypes.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// // a) Your answer: To be honest I have no clue but I'm guessing the number 4
+// // b) Verify and explain:Incorrect b/c the method typeof identifies the data type which is a number.
```

Comment:
First, thank you for your honestly. What's happening here is that typeof is giving us the data type of the array's length. You were close!